ingestion-beam/src/test/java/com/mozilla/telemetry/decoder/SanitizeAttributesTest.java

```diff
@@ -17,42 +18,82 @@
 
 public class SanitizeAttributesTest extends TestWithDeterministicJson {
 
+  static final byte[] EMPTY_JSON = "{}".getBytes(StandardCharsets.UTF_8);
+
   @Rule
   public final transient TestPipeline pipeline = TestPipeline.create();
 
   @Test
   public void testNoop() {
-    Map<String, String> attributes = ImmutableMap.<String, String>builder()
+    Map<String, String> inputAttributes = ImmutableMap.<String, String>builder()
         .put(Attribute.DOCUMENT_NAMESPACE, "test") //
         .put(Attribute.DOCUMENT_TYPE, "test") //
         .put(Attribute.DOCUMENT_VERSION, "1") //
         .put(Attribute.SUBMISSION_TIMESTAMP, "2021-01-01T12:00:00.123456Z") //
         .put(Attribute.GEO_CITY, "Toronto") //
         .build();
-    PCollection<String> result = pipeline //
-        .apply(Create.of(new PubsubMessage("{}".getBytes(StandardCharsets.UTF_8), attributes))) //
+
+    // Output should match the input exactly.
+    Map<String, String> expected = new HashMap<>(inputAttributes);
+
+    PCollection<Map<String, String>> result = pipeline //
+        .apply(Create.of(new PubsubMessage(EMPTY_JSON, inputAttributes))) //
         .apply(SanitizeAttributes.of("schemas.tar.gz"))
-        .apply(MapElements.into(TypeDescriptors.strings())
-            .via(message -> message.getAttribute(Attribute.SUBMISSION_TIMESTAMP)));
-    PAssert.that(result).containsInAnyOrder("2021-01-01T12:00:00.123456Z");
+        .apply(MapElements
+            .into(TypeDescriptors.maps(TypeDescriptors.strings(), TypeDescriptors.strings()))
+            .via(PubsubMessage::getAttributeMap));
+    PAssert.that(result).containsInAnyOrder(expected);
     pipeline.run();
   }
 
   @Test
   public void testSubmissionTimestampGranularity() {
     // Depends on custom doctype injected in bin/download-schemas
-    Map<String, String> attributes = ImmutableMap.<String, String>builder()
+    Map<String, String> inputAttributes = ImmutableMap.<String, String>builder()
         .put(Attribute.DOCUMENT_NAMESPACE, "test") //
         .put(Attribute.DOCUMENT_TYPE, "seconds") //
         .put(Attribute.DOCUMENT_VERSION, "1") //
-        .put("submission_timestamp", "2021-01-01T12:00:00.123456Z") //
```

Comment:
I don't have enough familiarity with the code to understand why this wasn't an `Attribute` before but it looks uniform now.

Reply:
This is just cleaning up an oversight. That value of the constant is equal to `"submission_timestamp"`.